pkg/kubelet/cm/container_manager.go

```diff
@@ -146,6 +146,9 @@ type ContainerManager interface {
 	// Updates returns a channel that receives an Update when the device changed its status.
```

Comment:
please reword the commit message to describe the change (you'r not adding just "e2e tests").
Probably got garbled in the rebase/squash.

Reply:
done!